offerItemImport/offerItemMapping/offerItemMappingHandler.go

```diff
@@ -24,6 +25,9 @@ func NewOfferItemMappingHandler(deps Deps) OfferItemMappingHandlerInterface {
 }
 func (oi *OfferItemMappingHandler) Run() error {
 	files := adapters.GetFiles(oi.sourcePath)
+	if len(files) == 0 {
+		return fmt.Errorf("please, put offer items file to %v", oi.sourcePath)
```

Comment:
I'd think about how to improve the error message

Reply:
need to discuss in future